migrations/5_test_case_id_type.up.sql

```diff
@@ -0,0 +1,8 @@
+ALTER TABLE test_item
+    ALTER COLUMN test_case_id TYPE VARCHAR(256);
```

Comment:
what would be with existed case ids? maybe you need to move them into hash?

Reply:
update logic fixed